com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/PyFunctionFilter/PyFunctionFilter_cpp.cgt

```diff
@@ -92,9 +95,18 @@ void MY_OPERATOR::process(Tuple const & tuple, uint32_t port)
 print splpy_inputtuple2value($pystyle);
 %>
 
+<%if ($pystyle eq 'spltuple') {%>
+@include "../pyspltuple2dict.cgt"
+  if (streamsx::topology::Splpy::pyTupleFilter(function_, pyDict)) {
+      submit(tuple, 0);
+  }
+  Py_DECREF(pyDict);
```

Comment:
This looks wrong, as it implies that we are holding the Python lock across the submit.

Also anything that deviates from the existing pattern makes me question why the approach is different.